wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/FileUtil.java

```diff
@@ -68,18 +68,17 @@ public static boolean validateFileType(final FileItemWrap newFile, final List<St
 			// If it exists
 			if (split.length == 2) {
 				// Then check if it matches supplied extension(s)
-				extMatch = fileExts.stream().anyMatch(ext -> ext.matches("(.*)" + split[1]));
+				extMatch = fileExts.stream().anyMatch(fileExt -> fileExt.equals("." + split[1]));
 			}
 		}
 		// If extension match is unsucessful, then move to fileMimes list
 		boolean mimeMatch = false;
```

Comment:
is the initial value strictly necessary?

Reply:
while strictly not necessary, if not initialised to `false`, I need to set to false 
* in `if` statement in `try` block,  
* `catch` block, 
* and add new else block after [line 93](https://github.com/BorderTech/wcomponents/blob/4e6e825546737773489ce40ae9f377734e0e50c4/wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/FileUtil.java#L93)

Reply:
ok